### ISSUE TITLE
Changing a label from an empty value to an empty value shows an error

### ISSUE THREAD

**Issue by Nukem9** [x64dbg/x64dbg#176]
```
---------------------------
Error!
---------------------------
DbgSetLabelAt failed!
---------------------------
OK
---------------------------
```

Same happens for comments.

----------------------------------------
- Bitbucket: https://bitbucket.org/mrexodia/x64_dbg/issue/176
- Originally reported by: [RaMMicHaeL](http://bitbucket.org/RaMMicHaeL)
- Originally created at: 2014-08-15T22:49:42.178

**Comment by Nukem9** [comment]
yea, this is the desired behavior. If you want to clear all comments use the so-called autocomments (comments not created by a user):

```
#!c++

BRIDGE_IMPEXP bool DbgSetAutoCommentAt(duint addr, const char* text);
BRIDGE_IMPEXP void DbgClearAutoCommentRange(duint start, duint end);
BRIDGE_IMPEXP bool DbgSetAutoLabelAt(duint addr, const char* text);
BRIDGE_IMPEXP void DbgClearAutoLabelRange(duint start, duint end);
```

This is the behavior of the comment functions:
```
#!c++
bool commentset(uint addr, const char* text, bool manual)
{
<cut>
    if(!*text) //NOTE: delete when there is no text
        return commentdel(addr); //commentdel simply fails if there is no comment
<cut>
}
```

----------------------------------------
Original comment by: [mrexodia](http://bitbucket.org/mrexodia)

**Comment by Nukem9** [comment]
I wasn't talking about the API, but about x64_dbg functionality. Click on an address which has no label, click ':', and then Enter. You will see an error message. Looks like a bug.

Perhaps `commentdel(addr)` should return true even if the label is already absent.

----------------------------------------
Original comment by: [RaMMicHaeL](http://bitbucket.org/RaMMicHaeL)

**Comment by Nukem9** [comment]
GUI: resolved issue #176 (empty label/comment shows error)


-> <<cset 10d098eaa008>>

----------------------------------------
Original comment by: [mrexodia](http://bitbucket.org/mrexodia)

**Comment by Nukem9** [comment]
DBG: resolved issue #176 (for real this time)


-> <<cset 6effe440a077>>

----------------------------------------
Original comment by: [mrexodia](http://bitbucket.org/mrexodia)

**Comment by Nukem9** [comment]
Revert "GUI: resolved issue #176 (empty label/comment shows error)"

This reverts commit 10d098eaa008adeffdf6bd2c6245e7554d244928.


-> <<cset 0a8c824e7848>>

----------------------------------------
Original comment by: [mrexodia](http://bitbucket.org/mrexodia)
